src/Compilers/CSharp/Test/Semantic/Semantics/NullableReferenceTypesTests.cs

```diff
@@ -1217,46 +1786,6 @@ static void Main()
             comp1.VerifyDiagnostics();
         }
 
-        [Fact]
-        public void NonNullTypes_False_Circular()
```

Comment:
>NonNullTypes_False_Circular [](start = 20, length = 27)

Are these tests no longer relevant?

Reply:
They weren't relevant with the design that NonNullTypes attribute must be synthesized, but isn't allowed in source. We just revised that decision. I'll add them back. Thanks

---
In reply to: [214447042](https://github.com/dotnet/roslyn/pull/29180#discussion_r214447042) [](ancestors = 214447042)